scripts/thanks.js

```diff
@@ -0,0 +1,33 @@
+if (!remoteUrl) {
+  var remoteUrl = "https://api.carpoolvote.com/live";
+}
+
+const handleBackendResponse = () => {
+  var data = tinyQuery.getAll();
+
+  var heading             = document.getElementById("thanks-header");
+  var introPara           = document.getElementById("introPara");
+  var successInfo         = document.getElementById("requestSuccessInfo");
+  var responseDetails     = document.getElementById("responseDetails");
+  var responseCode        = document.getElementById("responseCode");
+  var responseText        = document.getElementById("responseText");
+
+  var headingFailureText  = document.getElementById("sorryHeading").innerHTML;
+  var paraFailureText     = document.getElementById("sorryPara").innerHTML;
+
+  if (data.code != "0") {
+    heading.innerHTML   = headingFailureText;
+    introPara.innerHTML = paraFailureText;
+
+    successInfo.setAttribute("hidden", "hidden");
+    successInfo.setAttribute("style", "display:none");    
+
+    responseDetails.removeAttribute("hidden");
+    responseDetails.setAttribute("style", "");    
+
+    responseCode.innerHTML = data.code;
+    responseText.innerHTML = data.info;
+  }
+} 
```

Comment:
If we want to be this precise on JS style, I definitely recommend that the contributing guide specifies a linter (plus settings) or at minimum a link to some recommended style. I’m sure you are aware that there are various JS communities who take widely different views on this. Many of my fellow full-stack/NodeJs coders take the approach in https://standardjs.com (which is somewhat different).

Without any initial guidance from a contributing guide, someone who first encounters this guidance in a PR comment may feel it's both arbitrary and unwelcoming (and we especially need contributors with the above background to help us). My personal views are simply to avoid the main pitfalls and follow any provided contributing guidance.